### DISCUSSION TITLE
How to prevent specifying the same option twice

### DISCUSSION THREAD

**Discussion by babysnakes** [spectreconsole/spectre.console#305] [Q&A] [answered]
Hi,

(Sorry about the *free form* issue, This is not a bug and not a feature request - it's a question 😄)

It seems that by default I can specify an option multiple times and the last one is the one that counts (this is from the *Demo* example:

```
PROMPT > .\bin\Debug\net5.0\Demo.exe run --verbosity n --verbosity m
┌───────────────────┬─────────┐
│ Name              │ Value   │
├───────────────────┼─────────┤
│ Configuration     │ Debug   │
│ Framework         │ null    │
│ RuntimeIdentifier │ null    │
│ ProjectPath       │ null    │
│ LaunchProfile     │ null    │
│ NoLaunchProfile   │ False   │
│ NoBuild           │ False   │
│ Interactive       │ null    │
│ NoRestore         │ False   │
│ Verbosity         │ Minimal │
│ NoDependencies    │ False   │
│ Force             │ False   │
└───────────────────┴─────────┘
```

Is there a way to disallow it?

(sorry if I miss something obvious but I'm completely new to windows, .Net and F#).

Thanks

Haim

**Comment by phil-scott-78** [comment, marked as answer]
Right now there isn't a way to fail, but that behavior is pretty common for CLIs. E.g. here's `GH` 

![image](https://user-images.githubusercontent.com/2447331/111202477-48563880-859a-11eb-83ac-11c9c03f8efb.png)

and just to verify here's `git` on ubuntu

![image](https://user-images.githubusercontent.com/2447331/111203367-4e004e00-859b-11eb-80c0-87e6d0bc94c1.png)

**Comment by babysnakes** [comment]
Yeah, I can live with it. I just started playing with it and I'm not sure how it handles lists - many libraries handle lists by repeating the option so I wanted to differentiate ...

In any case, thanks for what seems to be a great library for displaying data in the console.

Haim
